nextflow.config

```diff
@@ -3,116 +3,151 @@
  *  nf-core/epitopeprediction Nextflow config file
  * -------------------------------------------------
  * Default config options for all environments.
- * Cluster-specific config options should be saved
- * in the conf folder and imported under a profile
- * name here.
  */
 
 // Global default params, used in configs
 params {
+  // Pipeline specific options
+  alleles = false
+  filter_self = false
+  mhc_class = 1
+  peptides = false
+  peptide_length = (mhc_class == 1) ? 11 : 16
+  tools = 'syfpeithi'
+  reference_genome = 'GRCh37'
+  somatic_mutations = false
+  wild_type = false
 
-  container = 'nfcore/epitopeprediction:latest' // Container slug. Stable releases should specify release tag!
+  // Additional annotation files
+  differential_gene_expression = false
+  gene_expression = false
+  gene_list = "$baseDir/assets/all_coding_genes_GRCh_ensembl_hgnc.tsv"
```

Comment:
is this specific to GRCh37 or GRCh38?

Reply:
Good point... I have to check but I guess it might be. In general I don't like how this is handled right now since there is no versioning and it might be outdated/different for future Ensembl releases. However, I didn't find a better solution so far.

Reply:
I'm guessing that can be added within AWS iGenomes in a specific folder or with a specific name to help with versioning, and then you just need to mention the right one in your config file, and easier to update.